CLAUDE.md

```
@@ -0,0 +1,147 @@
+**Native SDK Repositories**:
+- Android: https://github.com/CleverTap/clevertap-android-sdk
+- iOS: https://github.com/CleverTap/clevertap-ios-sdk
+- Web: https://github.com/CleverTap/clevertap-web-sdk
+
+This is a **wrapper SDK** - it doesn't implement analytics functionality directly but bridges Flutter to the native CleverTap SDKs on Android, Web and iOS.
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)